docker/configs/nginx/app.conf.prod

```diff
@@ -3,11 +3,11 @@
 #
 
 upstream app-server {
-        server icubam-server:8888;
+        server icubam_server:8888;
```

Comment:
Not really related to this PR, but it might have been more readable to define it as `icubam_www_server` to deflect its name in the code.

Reply:
Good point. I'll do the change. Thanks